frontend/cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should add baseurl variable to remove reuse of localhost:3000 for navigation in all tests

Reply:
Could also add watchForFileChanges=true to rerun the tests once a change has been made

Reply:
Default for watchForFileChanges is already true. I'll add baseUrl